.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Ensure that the tokens are not included in version control (e.g., .gitignore for git) to prevent unintended exposure of sensitive data.